src/application/common/commands-common.ts

```diff
@@ -8,6 +8,7 @@
 import { MAIN_MENU_BAR, MenuContribution, MenuModelRegistry } from './menu';
 import { CommandContribution, CommandRegistry } from './command';
 import { injectable } from "inversify";
+let Globalize = require("globalize");
```

Comment:
I have a question , If I put "import Globalize from 'Globalize', it does not work. 
I need to create a new file ( globalize.d.ts) and in there I would need to put  the following:
/// <reference types='globalize' />
/* tslint:disable */
declare namespace globalize { }
declare module 'globalize' {
    export default Globalize;
}

For now, should we just use "Let" and not create this dependencies to a new file ?